### PR TITLE
Added board files for Artisense reference design RD00

### PR DESCRIPTION
**Pull request by m-byte**
VID/PID as assigned by Espressif (see https://github.com/espressif/usb-pids)

### CODE REVIEW

File: ports/esp32s2/boards/artisense_rd00/board.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID           0x303A
+#define USB_PID           0x80B0
+#define USB_MANUFACTURER  "Artisense"
+#define USB_PRODUCT       "Reference Design RD00"
+
+#define UF2_PRODUCT_NAME  USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID      "AS-RD00"
```
Comment (hathach) [inline ports/esp32s2/boards/artisense_rd00/board.h:71]: `Board-ID` should follow the format of uf2 https://github.com/microsoft/uf2#files-exposed-by-bootloaders

```
The Board-ID field is machine-readable and consists of a number of dash-separated tokens. The first token is the CPU type, second is the board type, and third is the board revision. More tokens can be also added.
```

**Review by hathach — CHANGES_REQUESTED** [review]
all look good, except the board id must follow the sdd format from uf2 specs.

File: ports/esp32s2/boards/artisense_rd00/board.h (hunk 1 above)
Comment (m-byte) [reply ports/esp32s2/boards/artisense_rd00/board.h]: OK. Should be fixed now

**Review by hathach — APPROVED** [review]
look good now, thanks for your pr

**Comment by m-byte** [comment]
Thanks

**Pull request merged**